xapian-core/tests/run_glass

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+
+./runtest ./apitest -b glass
+./runtest ./apitest -b singlefile_glass
+./runtest ./apitest -b multi_glass
+./runtest ./apitest -b remoteprog_glass
+./runtest ./apitest -b remotetcp_glass
```

Comment:
By default, `sh` just continues when a command fails, and the non-zero return code is overwritten by the return code of the next command.  Here that means that only the success/failure of `./runtest ./apitest -b remotetcp_glass` will get reflected in the success/failure of the testsuite as a whole.

You can use `set -e` (best put right after the `#!` line) to tell the shell to exit if a command fails (there are some corner cases - for example, the condition of  an `if` or `while` failing sensibly won't cause the script to exit, but that's also true for commands inside a shell function called from the condition of an `if` or `while`, which tends to catch people out).

Demonstration (`$?` is 0 for success, non-zero if the command failed):

```
$ cat test.sh
#!/bin/sh
false
echo OK
$ ./test.sh; echo $?
OK
0
$ cat teste.sh 
#!/bin/sh
set -e
false
echo OK
$ ./teste.sh; echo $?
1
```

Reply:
Thanks. Good to know this. I pushed the change.